doc/user/content/sql/create-source/load-generator.md

```diff
@@ -209,6 +267,81 @@ SELECT * from bids;
  12 |  3338 |          1 |     97 | 2022-09-16 23:24:09.332+00
 ```
 
+### Creating a marketing load generator
+
+To create a load generator source that simulates an online marketing campaign:
+
+```sql
+CREATE SOURCE marketing
+  FROM LOAD GENERATOR MARKETING
+  FOR ALL TABLES
+  WITH (SIZE = '3xsmall');
+```
+
+To display the created subsources:
+
+```
+SHOW SOURCES;
+```
+```nofmt
+          name          |      type      | size
+------------------------+----------------+------
+ clicks                 | subsource      |
+ conversion_predictions | subsource      |
+ coupons                | subsource      |
+ customers              | subsource      |
+ impressions            | subsource      |
+ leads                  | subsource      |
+ marketing              | load-generator | 1
+ marketing_progress     | subsource      |
+```
+
+To find all impressions and clicks associated with a campaign over the last 30 days:
+
+```sql
+WITH
+    click_rollup AS
+    (
+        SELECT impression_id AS id, count(*) AS clicks
+        FROM clicks
+        WHERE click_time - INTERVAL '30' DAY <= mz_now()
+        GROUP BY impression_id
+    ),
+    impression_rollup AS
+    (
+        SELECT id, campaign_id, count(*) AS impressions
+        FROM impressions
+        WHERE impression_time - INTERVAL '30' DAY <= mz_now()
+        GROUP BY id, campaign_id
+    )
+SELECT campaign_id, sum(impressions) AS impressions, sum(clicks) AS clicks
+FROM impression_rollup LEFT JOIN click_rollup USING(id)
+GROUP BY campaign_id;
+```
+```nofmt
```

Comment:
The command and output are rendering as a single code block:
<img width="689" alt="Screenshot 2023-05-22 at 4 40 34 PM" src="https://github.com/MaterializeInc/materialize/assets/15893490/325eb917-3aa7-4702-b3a8-335ef93257d4">

Can you try to insert an empty line between then?